pkg/tagger/tagstore.go

```diff
@@ -131,31 +137,64 @@ func (e *entityTags) get(highCard bool) ([]string, []string) {
 	}
 
 	// Cache miss
-	var arrays [][]string
 	var sources []string
-	lowCardCount := 0
+	tagSortList := make(map[string][]tagSort)
```

Comment:
we're not sorting the tags, but de-duplicating them, could we use a more explicit name for both types?